kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-client-common/src/main/java/org/kie/workbench/common/stunner/core/client/shape/impl/NodeShapeImpl.java

```diff
@@ -66,5 +67,13 @@ public void applyPosition(final Node<View<W>, Edge> element,
     @Override
     public void applyState(final ShapeState shapeState) {
         getShape().applyState(shapeState);
+        ShapeView shape = getShapeView();
+        if (!isSelected() && shape instanceof HasControlPoints) {
+            ((HasControlPoints<Object>)shape).hideControlPoints();
```

Comment:
I think formatting doesn't follow the kiegorup one on this line.
```
((HasControlPoints<Object>) shape).hideControlPoints();
```

Reply:
True, thank you, good catch. Applied.